docs/source/inode.puml

```diff
@@ -70,6 +70,11 @@ class BufferDelayNode {
   +process_document()
 }
 
+class DenesterNode{
```

Comment:
Please could you also add the denester to the config `.rst` file that describes available nodes?